features/support/pages/admin/pending_projects_page.rb

```diff
@@ -0,0 +1,22 @@
+module Admin
+  # Encapsulates user interactions with the new project page
+  class PendingProjectsPage < SitePrism::Page
+    set_url "/admin/pending_projects"
+    sections :_pending_projects, PendingProjectSection, "*[data-type='pending-project']"
+    section :moderator_action_form, ModeratorActionFormSection, "*[data-type='moderator-action-form']"
+
+    def pending_projects
+      ElementCollection.new(_pending_projects)
+    end
+
+    def approve(project:, reason: "I just like it, OK?")
+      project_to_approve = pending_projects.element_for(project)
+      project_to_approve.begin_approval_button.click
+      moderator_action_form.submit(reason: reason)
+    end
+
+    def t(*args, **kwargs)
+      I18n.t(*args, **kwargs)
+    end
```

Comment:
We probably want to pull this up to our own `PageObject` base class, so i18n.t is easier.